dashboards/k6-prometheus/k6-prometheus.json

```diff
@@ -0,0 +1,629 @@
+{
+  "name": "k6 - Prometheus",
+  "description": "",
+  "pages": [
+    {
+      "name": "k6",
+      "description": null,
```

Comment:
Just a few more descriptions to fix here

Reply:
should be solved